tests/test_ppo_trainer.py

```diff
@@ -875,3 +875,38 @@ def test_generation(self):
         generations_single = tokenizer.batch_decode(generations_single)
 
         self.assertEqual(generations_single, generations_batched)
+
+    @unittest.skip("Fix by either patching `whomai()` to work in the staging endpoint or use a dummy prod user.")
+    def test_push_to_hub_if_best_reward(self):
+        REPO_NAME = "test-ppo-trainer"
+        repo_id = f"{CI_HUB_USER}/{REPO_NAME}"
+
+        dummy_dataset = self._init_dummy_dataset()
+
+        push_to_hub_if_best_kwargs = {"repo_id": repo_id}
+
+        ppo_config = PPOConfig(
+            batch_size=2,
+            mini_batch_size=1,
+            log_with=None,
+            push_to_hub_if_best_kwargs=push_to_hub_if_best_kwargs,
+            steps_beetwen_check=1,
```

Comment:
```suggestion
            compare_steps =1,
```
Fix typo and match your variable name :)